Sources/SymbolPicker/SymbolPicker.swift

```diff
@@ -88,7 +89,18 @@ public struct SymbolPicker: View {
     /// user-selected SFSymbol.
     /// - Parameter symbol: String binding to store user selection.
     public init(symbol: Binding<String>) {
+        _symbol = Binding(get: {
+            return symbol.wrappedValue
+        }, set: { newValue in
+            /// As the `canBeNone` is set to `false`, this can not be `nil`
+            symbol.wrappedValue = newValue!
```

Comment:
I know this could be pretty safe, but can we use `guard` instead of force unwrap to avoid potential crash?